packages/core/jest.config.js

```diff
@@ -19,7 +19,7 @@
 
 const baseConfig = require('../../jest.config.base');
 
-const {TextDecoder, TextEncoder} = require('util');
+const { TextDecoder, TextEncoder } = require('util');
```

Comment:
strange change here 🤔

Reply:
Nothing I did personally. Its prettier thats doing this for me